src/components/Accordion/components/ChevronDown.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+import { ChevronDownIcon } from '../../../icons';
+import { Colors } from '../../../essentials';
+
+export const ChevronDown = styled(ChevronDownIcon)`
+    color: ${props => (props.color ? props.color : Colors.AUTHENTIC_BLUE_900)};
```

Comment:
`props.color ? props.color : Colors.AUTHENTIC_BLUE_900)` can be simplified to `props.color || Colors.AUTHENTIC_BLUE_900)`